scalding-beam/src/main/scala/com/twitter/scalding/beam_backend/BeamOp.scala

```diff
@@ -183,7 +331,9 @@ object BeamOp extends Serializable {
     }
   }
 
-  case class KVToTuple[K, V](implicit kryoCoder: KryoCoder) extends PTransform[PCollection[KV[K, V]], PCollection[(K, V)]] {
+  case class KVToTuple[K, V](
```

Comment:
seems like this should take a `Coder[K]` and `Coder[V]` and build a `TupleCoder[K, V]` from those rather than use kryo for the whole thing. At least we can save the serialization tag that the value is a tuple, since we know that statically.